### DISCUSSION TITLE
Can't purchase a commerical license?

### DISCUSSION THREAD

**Discussion by samuelbraun04** [coqui-ai/TTS#3607] [General Q&A]
I'm trying to buy a commerical license for coqui but the link is inactive on https://coqui.ai/faq. Does anyone know where I could purchase it? Thank you!

**Comment by hrachkovinovoto** [comment]
> I'm trying to buy a commerical license for coqui but the link is inactive on https://coqui.ai/faq. Does anyone know where I could purchase it? Thank you!

https://coqui.ai/ if you visit the main site then you'll find that they've shutdown. No one really knows why they abandoned ship but hopefully someone else will bring forth an alternative that is truly open source and perhaps even let us train a new language by ourselves.
